install/kubernetes/cilium/charts/operator/templates/deployment.yaml

```diff
@@ -7,7 +7,14 @@ metadata:
   name: cilium-operator
   namespace: {{ .Release.Namespace }}
 spec:
+  # We support HA mode only for Kubernetes version > 1.14
```

Comment:
These changes should be squashed with the first commit.

Reply:
I don't get it. I didn't add this line.